hs_core/views/__init__.py

```diff
@@ -1032,6 +1032,39 @@ def get_metadata_terms_page(request, *args, **kwargs):
     return render(request, 'pages/metadata_terms.html')
 
 
+@login_required
+def get_user_data(request, user_id, *args, **kwargs):
+    """
+    This view function must be called as an AJAX call
+
+    :param user_id: id if the user for whom data is needed
+    :return:
+    """
+    user = utils.user_from_id(user_id)
+
+    if user.userprofile.middle_name:
+        user_name = "{} {} {}".format(user.first_name, user.userprofile.middle_name, user.last_name)
+    else:
+        user_name = "{} {}".format(user.first_name, user.last_name)
+
+    user_data = {'name': user_name, 'email': user.email}
+    user_data['url'] = '{domain}/user/{uid}/'.format(domain=utils.current_site_url(), uid=user.pk)
+    user_data['phone'] = user.userprofile.phone_1 if user.userprofile.phone_1 else ''
```

Comment:
I will check if there is a possibility of not having a value for phone_1 while a value for phone_2 can exist. then I would do what you have suggested.